ansible/files/sonic_example_ixia_devices.csv

```diff
@@ -0,0 +1,4 @@
+Hostname,ManagementIp,HwSku,Type
+example-ixia-1,10.0.0.20/32,IXIA-tester,DevIxiaChassis
+example-s6100-dut-1,10.0.0.10/32,Dell-S6100,DevSonic
+example-ixia-api-serv-1,10.0.0.30/32,IXIA-tester,DevIxiaApiServ
```

Comment:
What is this api server? Didn't see this specified in inventory 'example_ixia' or creategraph

Reply:
This is the ptf-ixia in the future. It needs to be there. We don't have code to deploy yet.